### PR TITLE
Add Amazon EventBridge to Slack Template

### PR DESCRIPTION
**Pull request by AdletMayerx**

### CODE REVIEW

File: events/amazon_eventbridge-http/amazon_eventbridge-slack.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[
+    {
+        "id": "20230407_1",
```
Comment (ehisakhile) [inline events/amazon_eventbridge-http/amazon_eventbridge-slack.json:3]: ID is not consistent with that of mock-up.ts

File: mock-up/mock.ts [hunk 2]
```diff
@@ -89,13 +89,23 @@
...
             "id": '20230406_2'
+},
+
+{
+            sinkIcon: "slack",
+            source: 'amazon-eventbridge',
+            sink: 'http',
+            "description": "Receive real-time notifications on a Slack Channel when someone uses Amazon S3 to create or delete Bucket.",
+            "subTitle": "Amazon EventBridge CreateBucket | DeleteBucket event triggers a Slack message",
+            "id": '20230406_2'
```
Comment (ehisakhile) [inline mock-up/mock.ts:108]: ID is not consistent with that of the template file
Kindly use current date as ID

**Comment by ehisakhile** [comment]
The wrong body format for Slack was used. 

Use this in the `body` format

```
"body": {
              "blocks": [
                  {
                    "type": "section",
                    "text": {
                      "type": "mrkdwn",
                      "text": "Hello World, This is where I put my text"
                    },
                    "accessory": {
                      "type": "image",
                      "image_url": "image_url",
                      "alt_text": "image_url_alt_text"
                    }
                  }
                ]
            }
```

**Comment by ehisakhile** [comment]
You need to modify
`"text": "Hello World, This is where I put my text"`, `"image_url": "image_url"` and `"alt_text": "image_url_alt_text"`

Look at the expected output of the template

**Comment by AdletMayerx** [comment]
> You need to modify `"text": "Hello World, This is where I put my text"`, `"image_url": "image_url"` and `"alt_text": "image_url_alt_text"`
> 
> Look at the expected output of the template

**Comment by AdletMayerx** [comment]
sorry, i don't understand where i need to look

**Review by ehisakhile — APPROVED** [review]
LGTM

**Pull request merged**